lib/Dialect/SPIRV/SPIRVDialect.cpp

```diff
@@ -267,18 +307,16 @@ Optional<Type> parseAndVerify<Type>(SPIRVDialect const &dialect, Location loc,
 }
 
 template <>
-Optional<spirv::StructType::LayoutInfo>
-parseAndVerify(SPIRVDialect const &dialect, Location loc, StringRef spec) {
+Optional<uint64_t> parseAndVerify(SPIRVDialect const &dialect, Location loc,
+                                  StringRef spec) {
   uint64_t offsetVal = std::numeric_limits<uint64_t>::max();
   if (!spec.consume_front("[")) {
     emitError(loc, "expected '[' while parsing layout specification in '")
         << spec << "'";
     return llvm::None;
   }
   if (spec.consumeInteger(10, offsetVal)) {
```

Comment:
This is not related to your change, but can you add spec.trim() before spec.consumeInteger?

Reply:
I'll update it.